types/messages.go

```diff
@@ -191,7 +191,7 @@ func (msg *SignedSSVMessage) Decode(data []byte) error {
 // - Data length should not be 0
 func (msg *SignedSSVMessage) Validate() error {
 	if msg.OperatorID == 0 {
-		return errors.New("OperatorID in SignedSSVMessage is 0")
+		return errors.New("signer ID 0 not allowed")
 	}
 	if len(msg.Signature) == 0 {
```

Comment:
is there a reason to not require the expected length of RSA signatures, or is that not the right place to do so, or not the right assumption for the spec to make?

Reply:
Actually, in the beginning, I wanted to let it open. But I've also added the custom decoding function to be used by implementation that sets it to 256 bytes
```go
signatureSize = 256
signature := encoded[signatureOffset : signatureOffset+signatureSize]
```
I have no opinion right now on whether we should check it here or leave the current check. 🤔

Reply:
@GalRogozinski any reason why we shouldn't check that `len(sig) == signatureSize` in spec?

Reply:
When you decode the data from pubsub you create a byte array of the expected length
maybe change `SignedMessage` to have `Signature [256]byte`

btw @MatheusFranco99 I remember we said the encoded length of the sig we use is actually greater correct?

Reply:
Ok, I'll change it to 256 bytes
This was about an encrypted key. For the signature, it's indeed 256